shell/server.go

```diff
@@ -438,14 +430,9 @@ func StartDocker(cfg *ProcessConfig, dockerRegistry, port, controller string) (*
 	}
 
 	// set the systemuser and password
-	unused := 0
-	systemUser := user.User{}
-	err = cp.GetSystemUser(unused, &systemUser)
-	if err != nil {
-		glog.Errorf("Unable to get system user account for client %s", err)
-	}
-	argv = append(argv, "-e", fmt.Sprintf("CONTROLPLANE_SYSTEM_USER=%s ", systemUser.Name))
-	argv = append(argv, "-e", fmt.Sprintf("CONTROLPLANE_SYSTEM_PASSWORD=%s ", systemUser.Password))
+	// FIXME: wtf?!?!?!
+	argv = append(argv, "-e", fmt.Sprintf("CONTROLPLANE_SYSTEM_USER=system_user "))
+	argv = append(argv, "-e", fmt.Sprintf("CONTROLPLANE_SYSTEM_PASSWORD= "))
```

Comment:
I think this may break the CC integration in RM. 
See https://github.com/zenoss/zenoss-prodbin/blob/develop/Products/ZenUtils/controlplane/__init__.py